saleor/graphql/shop/types.py

```diff
@@ -119,6 +120,11 @@ class Shop(graphene.ObjectType):
         required=False,
         description="Shipping methods that are available for the shop.",
     )
+    channel_currencies = graphene.List(
+        graphene.NonNull(graphene.String),
+        description="List of channel currencies.",
```

Comment:
I think this version is more descriptive, it should be easier to understand for new users, what do you think?
```suggestion
        description="List of all currencies supported by shop's channels.",
```